src/metabase/models/bookmark.clj

```diff
@@ -51,66 +52,72 @@
 
 (defn- bookmarks-union-query
   [user-id]
-  (let [as-null (when (= (mdb.connection/db-type) :postgres) (hx/->integer nil))]
+  (let [as-null (when (= (mdb.connection/db-type) :postgres) (h2x/->integer nil))]
     {:union-all [{:select [:card_id
                            [as-null :dashboard_id]
                            [as-null :collection_id]
                            [:card_id :item_id]
-                           [(hx/literal "card") :type]
+                           [(h2x/literal "card") :type]
                            :created_at]
-                  :from   [CardBookmark]
+                  :from   [:card_bookmark]
                   :where  [:= :user_id user-id]}
                  {:select [[as-null :card_id]
                            :dashboard_id
                            [as-null :collection_id]
                            [:dashboard_id :item_id]
-                           [(hx/literal "dashboard") :type]
+                           [(h2x/literal "dashboard") :type]
                            :created_at]
-                  :from   [DashboardBookmark]
+                  :from   [:dashboard_bookmark]
                   :where  [:= :user_id user-id]}
                  {:select [[as-null :card_id]
                            [as-null :dashboard_id]
                            :collection_id
                            [:collection_id :item_id]
-                           [(hx/literal "collection") :type]
+                           [(h2x/literal "collection") :type]
                            :created_at]
-                  :from   [CollectionBookmark]
+                  :from   [:collection_bookmark]
```

Comment:
It was sort of nice to be able to use the model name and not have to remember the table name (I was recently bitten by `User` being in `core_user`, not `user`, for example). But if this is how it has to be it's not a big deal.